torch/_inductor/debug.py

```diff
@@ -40,18 +42,17 @@
 
 
 @functools.lru_cache(None)
-def has_dot():
+def has_dot() -> bool:
     try:
         subprocess.check_output(["which", "dot"], stderr=subprocess.PIPE)
         return True
     except subprocess.SubprocessError:
         return False
 
 
-def draw_buffers(nodes, print_graph=False, fname=None):
+def draw_buffers(nodes: list[BaseSchedulerNode], print_graph=False, fname=None):
```

Comment:
nit: `List` is more idiomatic

Reply:
oh, I thought `list` / `from __future__ import annotations` was what the code should eventually look like

but I see from https://mail.python.org/archives/list/python-dev@python.org/message/VIZEBX5EYMSYIJNDBF6DMUMZOCWHARSO/ that there's actually some controversy over whether these PEPs are going to be made official as-is

I'll revert to use List